converter/src/main/java/gov/cms/qpp/conversion/validate/QualityMeasureIdValidator.java

```diff
@@ -305,7 +305,7 @@ protected void addMeasureConfigurationValidationMessage(Supplier<Object> check,
 			thoroughlyCheck(thisNode)
 					.incompleteValidation()
 					.singleValue(message, name);
-			return uuid.get().equals(thisNode.getValue(name));
+			return uuid.get().toString().equalsIgnoreCase(thisNode.getValue(name));
```

Comment:
why toString()

Reply:
You're right. Changing that.

Reply:
Oh I was looking at the wrong part of my code. The reason for that is because uuid.get() doesn't register as a string and needs to be a string in order to use the `equalsIgnoreCase` operation. I did change another piece of code to not use toString from this so thank you.

Reply:
The get() method from Supplier returns T and doesn't know it's a string. The only other way I could thing of is casting it as a string. Which could be better.

Reply:
I am trying an approach of changing the `Supplier` to be `<String>` instead of `<Object>`. We'll see how this fairs.

Reply:
Alright. Changed and let me know if it's better!... or worse. Thanks for pointing that out.

Reply:
Nice fix!